tests/unit/test_ops.py

```diff
@@ -99,7 +99,7 @@ def test_no_scipy(scm_run):
     with pytest.raises(
         ImportError, match="scipy is not installed. Run 'pip install scipy'"
     ):
-        scm_run.integrate()
+        scm_run.cumtrapz()
```

Comment:
Why the change here?

Reply:
`integrate` is deprecated. This removes a warning when running the test